pkg/services/ngalert/notifier/channels/opsgenie.go

```diff
@@ -80,17 +81,44 @@ func NewOpsgenieNotifier(model *NotificationChannelConfig, t *template.Template)
 	}, nil
 }
 
-// Notify sends an alert notification to Opsgenie
 func (on *OpsgenieNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
+	if success, errors := on.NotifyMultiple(ctx, as); !success {
```

Comment:
> I think this function signature can be simplified from (bool, []error) to just []error such that if no error occurred we can just return nil.

and then instead of checking for both `success` and then checking for non-nil values in `errors` we could just check that `err != nil`.